sites/search/search-en.html

```diff
@@ -0,0 +1,63 @@
+---
+{
+	"altLangPage": "search-fr.html",
+	"dateModified": "2023-06-29",
+	"description": "Documentation on how to use search bar.",
+	"language": "en",
+	"title": "GC Site Search Bar example"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<style>
+	.btn-square {
+		border-radius: 0;
+	}
+</style>
```

Comment:
What is that custom style for? We should not have any custom style. Regarding the boder radius, I do think we already have an CSS class that already do it.